src/routes.ts

```diff
@@ -458,12 +458,16 @@ export default function (
         console.error(err.stack);
       }
     }
-    if (env !== 'development') {
-      isExpected
-        ? res.status(err.status).json(err)
-        : res.status(500).json({ status: 500, error: 'Internal Server Error' });
+    if (config.handleError  === false) {
```

Comment:
This works as it should, but I'd like to have it a bit cleaner s.t. it works as expected in relation to other config settings ;)

- move the entry to `security` instead of making it a top-level config entry
- add the default explicitly to https://github.com/perfood/couch-auth/blob/master/src/config/default.config.ts and to https://github.com/perfood/couch-auth/blob/master/config.example.js
- make it work s.t. `undefined` corresponds to `false`. You could either set `true` as default or (which I'd prefer) rename it into `forwardErrors` which would then have `false` as default setting ;)